modules/gapi/include/opencv2/gapi/infer/onnx.hpp

```diff
@@ -122,6 +124,15 @@ template<typename Net> class Params {
         return *this;
     }
 
+    Params<Net>& cfgPostProc(const std::vector<cv::GMatDesc> &outs,
+                             const PostProc &pp,
+                             const std::vector<std::string> out_names) {
+        desc.out_metas = outs;
+        desc.custom_post_proc = pp;
+        desc.names_to_remap = out_names;
```

Comment:
`desc.names_to_remap = names_to_remap`, or something like this looks clearer

Reply:
Fixed.